proxy/installer/spacewalk-proxy-installer.spec

```diff
@@ -106,6 +106,9 @@ if [ -f /etc/sysconfig/apache2 ]; then
     sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES proxy_http
 fi
 sed -i -e"s/^range_offset_limit -1 KB/range_offset_limit none/" /etc/squid/squid.conf
+if ! grep pub\/repositories /etc/squid/squid.conf >/dev/null; then
+    sed -i 's;\(refresh_pattern /rhn/manager/download.*\);\1\nrefresh_pattern /pub/repositories/.*/repodata/.*$ 0 1% 1440 ignore-no-cache reload-into-ims refresh-ims;' /etc/squid/squid.conf
+fi
```

Comment:
Trying to confirm the correctness of such a sed command by looking at the code is not trivial :) But I assume you tested it and if it results in the line below contained in squid.conf, it is obviously correct as we are using the same rule for the other metadata as well. So fine with me.